src/internal.c

```diff
@@ -1429,9 +1429,9 @@ static const NameIdPair NameIdMap[] = {
 #ifndef WOLFSSH_NO_DH_GEX_SHA256
     { ID_DH_GROUP14_SHA256, "diffie-hellman-group14-sha256" },
 #endif
-#ifndef WOLFSSH_NO_ECDH_SHA2_NISTP256_KYBER_LEVEL1_SHA256
+#ifndef WOLFSSH_NO_ECDH_NISTP256_KYBER_LEVEL1_SHA256
     /* We use kyber-512 here to achieve interop with OQS's fork. */
-    { ID_ECDH_SHA2_NISTP256_KYBER_LEVEL1_SHA256, "ecdh-sha2-nistp256-kyber-512-sha256" },
+    { ID_ECDH_NISTP256_KYBER_LEVEL1_SHA256, "ecdh-nistp256-kyber-512-sha256" },
```

Comment:
`ecdh-nistp256-kyber-512-sha256` should become `ecdh-nistp256-kyber-512r3-sha256-d00@openquantumsafe.org` as per https://github.com/open-quantum-safe/openssh/issues/136